Description: Don't save bookmark thumbnails back to grilo, a bug there (fixed upstream but not releasable yet) means that this duplicates the bookmark.
Author: Iain Lane <iain.lane@canonical.com>
Forwarded: not-needed

--- totem-3.14.2.orig/src/icon-helpers.c
+++ totem-3.14.2/src/icon-helpers.c
@@ -195,7 +195,7 @@ thumbnail_media_async_thread (GTask    *
 	gnome_desktop_thumbnail_factory_save_thumbnail (factory, pixbuf, uri, g_date_time_to_unix (mtime));
 
 	/* Save the thumbnail URL for the bookmarks source */
-	save_bookmark_thumbnail (media, uri);
+	/* save_bookmark_thumbnail (media, uri); */
 
 	g_task_return_pointer (task, pixbuf, g_object_unref);
 	g_object_unref (task);
